helper/tests/acc/provisioners.go

```diff
@@ -68,6 +69,26 @@ func TestProvisionersAgainstBuilders(provisionerAcc ProvisionerAcceptance, t *te
 	}
 }
 
+// TestProvisionersPreCheck checks if the Provisioner with name is set in ACC_TEST_PROVISIONERS environment variable
+func TestProvisionersPreCheck(name string, t *testing.T) {
+	p := os.Getenv("ACC_TEST_PROVISIONERS")
+
+	if p == "all" {
+		return
+	}
+
+	provisioners := strings.Split(p, ",")
+	for _, provisioner := range provisioners {
+		if provisioner == name {
+			return
+		}
+	}
```

Comment:
The `strings.Contains(p,name)` like it was used before doesn't work here?

Reply:
While testing I found that the shell acceptance test were also running when trying to run powershell tests because `strings.Contains("powershell", "shell")` returns true so I went for the same logic used in the checkBuilders func to ensure an exact match.

Reply:
wow!! good catch! I was never going to think about this without seeing it happen! 👍